lib/screenplay/suppressed_predictions.ex

```diff
@@ -38,6 +41,76 @@ defmodule Screenplay.SuppressedPredictions do
     |> Repo.all()
   end
 
+  @doc """
+  Get all suppressed predictions modified for transit data
+  """
+  @spec get_all_suppressed_predictions_transit_data() :: [SuppressedPrediction.t()]
+  def get_all_suppressed_predictions_transit_data do
+    places = Places.get()
+
+    SuppressedPrediction
+    |> Repo.all()
+    |> Enum.map(fn
+      %SuppressedPrediction{
+        route_id: "Green",
+        location_id: location_id,
+        direction_id: direction_id
+      } ->
+        # Go through Green Line route and repopulate the appropriate branches with the unique direction id
+        Enum.find(places, fn place -> place.id == location_id end).routes
+        |> Enum.filter(fn route -> String.contains?(route, "Green") end)
+        |> Enum.map(fn route ->
+          if String.contains?(route, "Green") do
+            %{
+              stop_id: location_id,
+              route_id: route,
+              direction_id: direction_id
```

Comment:
The current data feed from Signs UI generates all possible entries, and includes a flag specifying whether each one is suppressed. I think the goal there was to always make the payload the maximum size, so there wouldn't be any surprise performance issues (on either end) if OIOs suddenly suppressed a lot of things. I think we probably want to keep that approach for this new API.